convert2rhel/actions/pre_ponr_changes/transaction.py

```diff
@@ -27,7 +27,6 @@ class ValidatePackageManagerTransaction(actions.Action):
     id = "VALIDATE_PACKAGE_MANAGER_TRANSACTION"
     dependencies = (
         "INSTALL_RED_HAT_GPG_KEY",
-        "REMOVE_EXCLUDED_PACKAGES",
```

Comment:
Maybe I would add some comment, that there is an implicit dependency of removing the special packages (as in `pre_ponr_changes/subscription.py`). Or needs to be it removed?